doc/src/hugo/content/guides/authors/design/patterns/command-handler-messages.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Command/Event Patterns"
+description: "Alternatives & Recommendations for Command Handler Messages"
+date: 2022-02-25T10:50:32-07:00
+draft: "false"
+weight: 10
+---
+
+## Introduction To Messages
+If you are not familiar with the concept of `Messages` as it pertains to RIDDL, see [here](../../../../../concepts/message/).
```

Comment:
I'm trying not to propagate these kinds of links any more because they are not verified by a hugo build when we check the documentation. Please change these to use the `relref` shortcode